src/popup/extract_path.js

```diff
@@ -2,20 +2,31 @@ var Popup = Popup || {};
 
 // Extract the relevant path from a location, such as `/foo` from URLs like
 // `www.gov.uk/foo` and `www.gov.uk/api/content/foo`.
-Popup.extractPath = function(location) {
+Popup.extractPath = function(location, renderingApplication) {
```

Comment:
I'm wondering if this does what we want it to. If the user switches between staging / production on a specific smart answer question page, will they be redirected to the landing page?